### DISCUSSION TITLE
Led

### DISCUSSION THREAD

**Discussion by Nikolosion** [Fishwaldo/sophgo-sg200x-debian#43] [Q&A]
Hello, how do turn off the flashing LED? Thanks in advance

**Comment by luvocean1** [comment]
I too am wondering.....the standar /mnt/system/ script does not exist....so what is driving the led?

**Reply by TheFlemoid** [reply]
Copying an answer I made in [another issue](https://github.com/Fishwaldo/sophgo-sg200x-debian/issues/44) about this topic:

> I was wondering the same thing, and Fishwaldo himself set me straight over on the Milk-V forum.
> 
> So the red LED is a power indication and isn't controlled by anything other then VCC being present, so you won't be able to control that one. The blue LED that flashes by default is controllable however (as well as the green and orange lights on the NIC if you wanted to control those).
> 
> The image is using native kernel LED support, which is discussed here: https://www.kernel.org/doc/html/latest/leds/leds-class.html
> 
> A more direct answer to your question, you can stop the LED from blinking with the command:
> 
> echo none > /sys/class/leds/blue/trigger
> 
> The blue LED should stop blinking, but it will probably be on. You can turn then turn the LED off with the command:
> 
> echo 1 > /sys/class/leds/blue/brightness
